rust/src/headless.rs

```diff
@@ -99,6 +99,10 @@ pub fn shutdown() {
     unsafe { binaryninjacore_sys::BNShutdown() };
 }
 
+pub fn is_shutdown_requested() {
+    unsafe { binaryninjacore_sys::BNIsShutdownRequested() };
```

Comment:
`BNIsShutdownRequested` returns `bool`
```suggestion
pub fn is_shutdown_requested() -> bool {
    unsafe { binaryninjacore_sys::BNIsShutdownRequested() }
```